name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/LGTBankPDFExtractor.java

```diff
@@ -125,10 +125,17 @@ private void addDividendeTransaction()
                 })
     
                 // Ex-Datum 12. Mai 2020
-                .section("date")
-                .match("^(Ex-Datum) (?<date>\\d+. \\w+ \\d{4})$")
+                .section("exDate")
```

Comment:
// Ex-Datum 12. Mai 2020
not the same for regEx
`^(Ex-Datum) (?<exDate>\\d+. \\w+ \\d{4})$`